internal/testcase/testcase.go

```diff
@@ -76,12 +76,18 @@ type TestCaseSet struct {
 	ExpectedEvents []logstash.Event `json:"expected" yaml:"expected"`
 
 	// ExportMetadata controls if the metadata of the event processed
-	// by Logstash is returned The metadata is contained in the field
+	// by Logstash is returned. The metadata is contained in the field
 	// `[@metadata]` in the Logstash event.
 	// If the metadata is exported, the respective fields are compared
 	// with the expected result of the testcase as well. (default: false)
 	ExportMetadata bool `json:"export_metadata" yaml:"export_metadata"`
 
+	// ExportOutputs controls if the ID of the output, a particular event has
+	// emitted by, is kept in the event or not.
+	// If this is enabled, the expected event needs to contain a field named
+	// __lfv_out_passed which contains the the ID of the Logstash output.
```

Comment:
s/the the/the/

I'll squash your first two commits ("Fix multiple outputs..." and "Typo") and fix this while I'm at it.